src/transformStateWithClones.js

```diff
@@ -7,7 +7,27 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  const states = [];
```

Comment:
Consider initializing `states` with the initial `currentState` to ensure the initial state is included in the returned array, especially if no actions are provided.